modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/account_selector/views/OrdersListView.js

```diff
@@ -100,7 +101,10 @@ function OrdersListView({
 					displayType="primary"
 					onClick={() =>
 						showOrderTypeModal
-							? Liferay.fire(OPEN_MODAL, {id: 'add-order-modal'})
+							? Liferay.fire(OPEN_MODAL, {
```

Comment:
Did you have the chance to test the behavior with 2 instances of the same portlet on the page?